docs/api.md

```diff
@@ -264,6 +264,7 @@ await context.close();
 
 <!-- GEN:toc -->
 - [event: 'close'](#event-close)
+- [event: 'pageevent'](#event-pageevent)
```

Comment:
I don't like the name `pageevent`. This looks to me like something happened inside the page. What about `newpage` or `pagecreated`?

Reply:
Renamed to 'page'